brands/shop/hifi.json

```
@@ -43,5 +43,15 @@
       "name": "JBL",
       "shop": "hifi"
     }
-  }
-}
\ No newline at end of file
+  },
+  "shop/hifi|Richer Sounds": {
+    "countryCodes": ["gb"],
+    "tags" : {
+      "brand": "Richer Sounds",
+      "brand:wikidata": "Q7330409",
+      "brand:wikipedia": "en:Richer Sounds",
+      "name": "Richer Sounds",
+      "shop": "hifi"
+    }
+  },
```

Looks like the build is failing because of this extra trailing comma..
```suggestion
  }
```